whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -1563,6 +1563,79 @@ public void lookup_inetnum_managed_attributes_resource_holder_abuse_contact() {
         assertThat(response.getWhoisObjects().get(0).getAttributes().get(5).getManaged(), is(true));    // source
     }
 
+    @Test
+    public void validate_country_managed_attribute_only_for_organisation() {
+        databaseHelper.addObject(
+                "mntner:       RIPE-NCC-HM-MNT\n" +
+                        "source:   TEST");
+        databaseHelper.addObject(
+                "organisation: ORG-TO1-TEST\n" +
+                        "org-name:     Test Organisation\n" +
+                        "org-type:     LIR\n" +
+                        "country:      NL\n" +
+                        "abuse-c:      TR1-TEST\n" +
+                        "source:       TEST");
+        databaseHelper.addObject(
+                "inetnum:       10.0.0.0 - 10.0.0.255\n" +
+                        "status:   ALLOCATED PA\n" +
+                        "org:      ORG-TO1-TEST\n" +
+                        "country:  NL\n" +
+                        "mnt-by:   OWNER-MNT\n" +
+                        "mnt-by:   RIPE-NCC-HM-MNT\n" +
+                        "source:   TEST");
+
+
+        final WhoisResources searchOrg = RestTest.target(getPort(), "whois/test/organisation/ORG-TO1-TEST?managed-attributes")
+                .request(MediaType.APPLICATION_XML_TYPE)
+                .get(WhoisResources.class);
+
+        assertThat(searchOrg.getWhoisObjects(), hasSize(1));
+        assertThat(searchOrg.getWhoisObjects().get(0).getPrimaryKey().get(0).getValue(), is("ORG-TO1-TEST"));
+        assertThat(searchOrg.getWhoisObjects().get(0).isManaged(), is(true));
+        assertThat(searchOrg.getWhoisObjects().get(0).getAttributes().get(3).getName(), is("country"));
+        assertThat(searchOrg.getWhoisObjects().get(0).getAttributes().get(3).getManaged(), is(true));
+
+
+        final WhoisResources searchinetnum = RestTest.target(getPort(), "whois/test/inetnum/10.0.0.0%20-%2010.0.0.255?managed-attributes&resource-holder&abuse-contact")
+                .request(MediaType.APPLICATION_XML_TYPE)
+                .get(WhoisResources.class);
+
+        assertThat(searchinetnum.getWhoisObjects(), hasSize(1));
+        assertThat(searchinetnum.getWhoisObjects().get(0).getPrimaryKey().get(0).getValue(), is("10.0.0.0 - 10.0.0.255"));
+        assertThat(searchinetnum.getWhoisObjects().get(0).getAttributes().get(3).getName(), is("country"));
+        assertThat(searchinetnum.getWhoisObjects().get(0).getAttributes().get(3).getManaged(), is(nullValue()));
+    }
+
+    @Test
+    public void update_orgnasation_country_code_fails() {
```

Comment:
typo: orgnasation -> organisation

This test is for testing the validation of country code updating only being allowed by RIPE NCC, not for the attribute being flagged as managed?

Reply:
I see the test case tests specifically that country code value can only be changed by RIPE NCC.
Can you add a test also where you do authenticate as RIPE and change the attribute value?

Reply:
There is separate test case to test that country code is shown as managed for organisation and not as managed attribute for other object type.

Reply:
I will add a test case that country code can be updated using overrides

Reply:
Done